test/http.spec.js

```diff
@@ -0,0 +1,29 @@
+import nock from 'nock';
+import path from 'path';
+import downloadFile from '../src/download-file';
+import fs from 'fs';
+import expect from 'expect';
+
+nock('http://local.foo.com')
```

Comment:
I was thinking in something that receives an array of images and creates all the links or something like that, and returns the nock server if you want to do something else with it. Or maybe a function that receives an array and the nock instance and adds the required endpoints.